test/integration/change-streams/change_stream.test.ts

```diff
@@ -1990,4 +1992,123 @@ describe('Change Streams', function () {
         .toJSON()
     )
     .run();
+
+  describe.only('BSON Options', function () {
+    let client: MongoClient;
+    let db: Db;
+    let collection: Collection;
+    let cs: ChangeStream;
+    beforeEach(async function () {
+      client = await this.configuration.newClient({ monitorCommands: true }).connect();
+      db = client.db('db');
+      collection = await db.createCollection('collection');
+    });
+    afterEach(async function () {
+      db.dropCollection('collection');
```

Comment:
`dropCollection` returns a promise, so we need to `await` it here as well

Reply:
done